tgui/packages/tgui/interfaces/Keybinds/Footer.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author Garash2k (https://github.com/garash2k)
+ * @license MIT
+ */
+import { Button } from 'tgui-core/components';
+
+import { useBackend } from '../../backend';
+import { KeybindsData } from './types';
+
+export const Footer = () => {
+  const { act, data } = useBackend<KeybindsData>();
+  const { hasChanges, resetting } = data;
+
+  return (
+    <>
+      <Button
+        onClick={() => act('confirm')}
+        color={!hasChanges ? null : 'good'}
```

Comment:
Verify, but I think `undefined` is preferred to `null` for when we're not changing the color.